src/contents/all/components/knot/Knot/Knot.tsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
 import Moment from 'react-moment';
 
-import { Timeline } from 'antd';
+import { Timeline, Icon, Popconfirm, DatePicker, Tooltip } from 'antd';
 
 import './Knot.scss';
 import '../Trix.scss';
 
+// <div className="knotEditorWrapper">
+//   <div className="knotEditor" ref={knotEditor}>
+//     {show &&
+//       <RichTextEditor
+//         autoFocus
+//         toolbarConfig={{ display: [] }}
+//         value={body}
+//         onChange={onChange}
+//         placeholder={hasKnots ? "Add another note..." : "Add a note..."}
+//       />
+//     }
+//   </div>
+// </div>
+
 const Knot = props => (
   <Timeline.Item className={`knot ${props.isLast ? "ant-timeline-item-last" : ""}`}>
     <div className={`knotCard ${props.knot.private ? "private" : ""}`}>
```

Comment:
wondering if we should replace "trix-content" with another instance of RichTextEditor so user can edit old knots directly? assuming there's a cleaner way of doing this (and maybe shouldn't even happen in Knot.tsx?)

Reply:
Yea this makes sense to me, I think your intuition is in the right place. We could probably separate out editing into it's own component, but we might as well leverage `KnotInput`, and just pass it a prop of the knot. Similar to mainframe where I have KnotEditor and pass in a Knot which gives it different functionality (edit vs create).

But if you think it's cleaner to create a separate component that leverages RichTextEditor for just editing, that's solid too.